crates/benches/runner/Cargo.toml

```diff
@@ -6,9 +6,20 @@ version.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+runner-macro = { path = "./runner-macro" }
```

Comment:
@matzayonc my intention here wasn't to move the katana-runner and macro here entirely. I think it still makes sense to live in the katana crate. Rather, we should refactor the sozo logic out of it, into a similar sozo runner crate, rather than including that inside the katana runner. Could you please make these changes and commit them as part of a separate PR?

Reply:
Sorry, I actually refactored it based on telegram conv, without reading this message, and included it in this PR already.